Tests/Functional/Hooks/class.tx_multicolumn_tcemainTest.php

```diff
@@ -159,4 +160,37 @@ public function copyContainerAndChildrenToOtherPageInDefaultLanguage()
         $this->assertSame(1, $count);
     }
 
+    /**
+     * create an new Multicolumn container
+     *
+     * @test
+     */
+    public function createNewContainerWithDefaultLanguage()
+    {
+        $config = [
+            'header' => 'Mutlicolumn insert Test',
+            'colPos' => 0,
+            'CType' => self::CTYPE_MULTICOLUMN,
+            'sys_language_uid' => 0,
+            'tx_multicolumn_parentid' => 0
```

Comment:
Please remove this -> should be either set by the hook or fallback to default field configuration